File: IQClientLib/Responses/Consumption/Consumption.cs
namespace IQClientLib.Responses.Consumption
{
    public class Consumption 
    {
        public int Id { get; set; }
        public int createdAt { get; set; }
        public string reportType { get; set; }
        public Cumulative cumulative { get; set; }
        public List<Line> lines { get; set; }
    }

    }

File: IQClientSite/ClientApp/src/app/IQResponses/Consumption/Consumption.ts
import { Cumulative } from "./Cumulative";
import { Line } from "./Line";



export class Consumption {
  id:number;
  createdAt: number;
  reportType: string;
  cumulative: Cumulative;
  lines: Line[];
}

export class ConsumptionDb extends Consumption {
  iqId: number;
}


